README.dev.md

```diff
@@ -6,11 +6,612 @@ You will need a machine with at least 2GB of RAM available (see
 https://github.com/zulip/zulip/issues/32 for a plan for how to
 dramatically reduce this requirement).
 
-Start by cloning this repository: `git clone https://github.com/zulip/zulip.git`
+Contents:
+
+* [First-time contributors start here](#first-time-contributors-start-here)
```

Comment:
What do you think about calling this "Development environment setup for first-time contributors" ?